css/styles.css

```
@@ -190,3 +216,61 @@ Colors:
 .btn-white:active {
   background-color: rgba(255, 255, 255, 0.35);
 }
+
+/* ***************** */
+/* About */
+/* ***************** */
+.about {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 71rem;
+  padding-left: 26rem;
+  padding-right: 24rem;
+  background-image: linear-gradient(
+      to right bottom,
+      rgb(255 255 255 /14%),
+      rgb(0 0 0 /10%)
+    ),
+    url("../img/bg-about.png");
+  box-shadow: inset 0 0 2.4rem 1.2rem rgb(0 0 0 /6%);
+}
+
+.about-content {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  flex: 1;
+  padding-bottom: 2rem;
+}
+
+.about-heading {
+  font-family: "Yeseva One";
+  font-size: 3.6rem;
+  color: #cc9900;
+}
+
+.about-text {
+  font-family: "Lato";
+  font-size: 1.6rem;
+  line-height: 1.4;
+  color: #4c4c4c;
+  padding-left: 2.4rem;
+  padding-right: 2.4rem;
+  text-align: center;
+}
+
+.cook {
+  width: 21.9rem;
+  height: 22.1rem;
+  transform: translateX(3.6rem);
+}
+
+.about-img-container {
+  padding-top: 3rem;
+}
+
+.about-img {
+  width: 62.1rem;
+  height: 62.5rem;
+}
```

It'll also work for the case
```suggestion
    margin-top: 3rem;
}
```